src/structures/ReactionCollector.js

```diff
@@ -13,6 +13,7 @@ const { Events } = require('../util/Constants');
 
 /**
  * Collects reactions on messages.
+ * Will automatically stop if the Message, Channel, or Guild are deleted.
```

Comment:
Lowercase and name the reasons, as they are not publicly documented yet.
```suggestion
 * Will automatically stop if the message (`'messageDelete'`), channel (`'channelDelete'`), or guild (`'guildDelete'`) are deleted.
```